content/800-data-platform/05-data-browser.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Data Browser'
+metaTitle: 'Data Browser'
+metaDescription: 'Data Browser'
+toc: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+Use the **Data Browser** in the Prisma Data Platform to view and edit your application data.
+
+</TopBlock>
+
+## Models (Tables or Collections)
+
+When you first open the **Data Browser**, you can review the tables in your application database which are based on the models defined in the Prisma schema file.
+
+<img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-models-view_zs1ihu.png" alt="Data Browser - Models view"/>
+
+You can select a table and its data opens in a new tab.
```

Comment:
This is a comment that applies to all the content here:
In the past we tried to use Prisma terms when talking about Prisma, and only fall back to the database terms when talking explicitly about the database. Here for example one could have a headline "Models" and then just talk about how you can see all your models and their data.

Reply:
Also need a comma after "database", before "which". :-)

Reply:
Thanks for the feedback, Jan. I'll fix all of these.

Reply:
So I did. And added a thing of my own.

I've added 3 admonitions defining `model`, `field`, `record` right at the beginning of those sections and connected those to their relational db and MongoDB counterparts. Also added links to Prisma ORM docs to cross-link to the context where these are defined.

I am obviously new to Prisma still and if it makes sense to cater to this need, we could leave them there. Or redo them in a more meaningful way.